showkase/src/main/java/com/airbnb/android/showkase/ui/ShowkaseComponentDetailScreen.kt

```diff
@@ -3,49 +3,23 @@ package com.airbnb.android.showkase.ui
 import android.content.Context
 import android.content.res.Configuration
 import androidx.compose.foundation.clickable
-import androidx.compose.foundation.layout.Arrangement
-import androidx.compose.foundation.layout.Column
-import androidx.compose.foundation.layout.Row
-import androidx.compose.foundation.layout.fillMaxWidth
-import androidx.compose.foundation.layout.height
-import androidx.compose.foundation.layout.padding
-import androidx.compose.foundation.layout.size
-import androidx.compose.foundation.layout.sizeIn
-import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.*
```

Comment:
Avoid wildcard imports. Might need to update you Android Studio settings since its probably happening due to default values present locally

Reply:
Looks like detekt also flagged this on CI, hence the build failure @marcorighini